### ISSUE TITLE
Storage Summary for Admins

### ISSUE THREAD

**Issue by LoXan007** [Chocobozzz/PeerTube#4304]
Hi,

first some backround: I run a instance on a hetzner server via yunohost and for me as admin it's important to have a overview how many available storage space i have and how my storage space is already used by the local hosted videos. Currently i need to login into the server everytime to see this cause the peertube admin area doesn't give me these information.

So my feature request(s) for admins are the following:
1. Ability to see the video size of every local video on my instance (after transcoding!)
2. Ability to show a summary which give the following informations: 
   -> Total available storage space for videos
   -> Currently used storage space from videos
   -> Free storage space for videos
So in summary: A Overvíew over storage usage of a instance.

I'm hoping that i make my feature requests clear enough and i would be happy if the mentioned possibilites could be implemented in the future Peertube version.

**Comment by Chocobozzz** [comment]
Merging this issue in https://github.com/Chocobozzz/PeerTube/issues/1888
